src/screens/ExplainSheet.js

```diff
@@ -257,6 +263,39 @@ export const explainers = (params, colors) => ({
     text: VERIFIED_EXPLAINER,
     title: lang.t('explain.verified.title'),
   },
+  unverified: {
+    extraHeight: 120,
+    emoji: '⚠️',
+    buttonText: lang.t('button.continue'),
+    buttonColor: colors?.blueGreyDark60,
+    secondaryButtonColor: colors?.alpha(colors?.appleBlue, 0.04),
+    secondaryButtonTextColor: colors?.appleBlue,
+    secondaryButtonText: lang.t('button.go_back_lowercase'),
+    title: lang.t('explain.unverified.title', {
+      symbol: params?.asset?.symbol,
+    }),
+    stillCurious: (
+      <Text {...getBodyTextPropsWithColor(colors)}>
+        {lang.t('explain.unverified.fragment1')}
+        <Text
+          color={colors?.appleBlue}
+          lineHeight="loose"
```

Comment:
removing this line fixes most of the lineHeight problem on android
![image](https://user-images.githubusercontent.com/12115171/176580094-19661e09-75c7-4370-a7ba-022f9ef58b01.png)

but yeah i see the text lineHeight is not consistent